build/targets/KoreBuild.SolutionBuild.targets

```diff
@@ -290,6 +290,8 @@ Runs the VSTest on all projects in the ProjectsToTest itemgroup.
 
   <PropertyGroup>
     <VSTestLogger Condition=" '$(VSTestLogger)' == '' AND '$(TEAMCITY_VERSION)' != '' ">trx</VSTestLogger>
+    <!-- ensure AppVeyor's auto test reporter remains enabled -->
+    <VSTestAutoReporters Condition="'$(APPVEYOR_API_URL)' != ''">true</VSTestAutoReporters>
```

Comment:
I added this as an escape hatch in case we want to force the TeamCity auto-reporter to turn on without needing to make changes KoreBuild.